### PR TITLE
js-feat-code

### PR DESCRIPTION
**Pull request by sampoonachot**

### CODE REVIEW

**Comment by rawfalafel** [comment]
Lmk if this is ready for review. It seemed like you opened this to help me debug an issue earlier.

**Comment by sampoonachot** [comment]
@rawfalafel ready for review

**Comment by sampoonachot** [comment]
@rawfalafel ready for review

File: client/styling/code.less [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        border-left: 3px solid @gray5;
+        display: inline-block;
+        padding: 10px 15px;
+        font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+        font-size: 12px;
+        line-height: 35px;
+        overflow: scroll;
+        white-space: nowrap;
+
+        &-is-full-width {
```
Comment (rawfalafel) [inline client/styling/code.less]: If the text always has this "full width" modifier when the `copy-command` has the "full-width" modifier, we usually move this rule inside `&-is-full-width` to reduce the number of classes used in the HTML.

File: client/styling/json-view.less [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            cursor: pointer;
+
+            &:before {
+                content: "\25B6";
+                position: absolute;
+                margin-left: -10px;
+                margin-top: -1px;
+            }
+
+            &-is-open {
```
Comment (rawfalafel) [inline client/styling/json-view.less]: Did you mean to nest this inside the "-is-collapsible" block?

Comment (sampoonachot) [inline client/styling/json-view.less]: not sure, I copied this from the design toolkit bem component folder. but i moved it outside "-is-collapsible"

**Pull request merged**